Chirp/src/Chirp.Web/Pages/BindingModels/MessageModel.cs

```diff
@@ -0,0 +1,8 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace Chirp.Web.Pages.BindingModels;
+
+public class MessageModel {
+    [BindProperty]
+    public string Message {get; set;}
```

Comment:
There shouldn't be a waning. Recommendation would be to declare the property as nullable for now.

Reply:
I agree and, have made the change.